src/pages/ChooseArea.svelte

```diff
@@ -157,6 +158,9 @@
     <SecondaryButton on:click={() => (showAbout = !showAbout)}
       >About</SecondaryButton
     >
+    {#if errorMessage}
+      <ErrorMessage bind:errorMessage />
```

Comment:
Just `{errorMessage}` is fine. `bind` is for two-way binding. Since `ErrorMessage` doesn't mutate this, we don't need two-way -- just pass in a regular prop
https://svelte.dev/tutorial/component-bindings

Reply:
Yeah I think I tried that at first but got red squiggles so did this for now, but maybe I got it slightly wrong somehow

Reply:
Just tried it again, weirdly the red squiggle decides that error is a boolean. Even weirder, that becomes true.

I'll post screenshot of it with bind, and then without:
![image](https://github.com/acteng/atip/assets/133757676/3929b56b-5ce3-4609-ae2c-086fe86390a7)


without:
![image](https://github.com/acteng/atip/assets/133757676/5ef7d83b-651a-4466-8ad7-cc4c2005c9cc)

Reply:
Can probably resolve with black magic reactive statements

Reply:
Even with the braces? `<Component errorMessage>` means "set errorMessage=true" and `Component {errorMessage}>` means "set errorMessage prop equal to the errorMessage variable"

Reply:
https://svelte.dev/repl/72808c5cdce14fc6862e1661b68d8809?version=3.22.2

Reply:
Sorry saw this during standup and decided not to respond at that moment. This resolves it yes cheers!